recipes/dd-agent.rb

```diff
@@ -105,6 +105,7 @@ def template_vars # rubocop:disable Metrics/AbcSize
 service_provider = nil
 if node['datadog']['agent6'] &&
    (((node['platform'] == 'amazon' || node['platform_family'] == 'amazon') && node['platform_version'].to_i != 2) ||
+   (node['platform'] == 'ubuntu' && node['platform_version'].to_f == 14.04) ||
```

Comment:
* Unfortunately this logic is currently duplicated in `https://github.com/DataDog/chef-datadog/blob/master/providers/monitor.rb` (twice), so you also need to update it there.
* Ubuntu switched to systemd in `15.04`, what do you think of comparing with `< 15.04` instead? (should work since we're comparing floats)